auth/README.md

```diff
@@ -258,6 +286,8 @@ startActivityForResult(
     RC_SIGN_IN);
 ```
 
+##### Phone number auth customization
```

Comment:
What about `authentication` instead of this shortcut?

Reply:
SGTM